src/redux/reducers/todosReducer.js

```diff
@@ -49,7 +67,19 @@ export default function todosReducer(state = initialState, action) {
         todoItems: updatedTasks,
       }
     
+    case UPDATE_TODO:
+      const { updatedTodo ,taskTitle} = action.payload
```

Comment:
Aren't we using Preetier in our project? 
```suggestion
      const { updatedTodo, taskTitle} = action.payload
```